packages/forma-36-react-components/src/components/Tag/Tag.tsx

```diff
@@ -32,7 +33,7 @@ const statusTagTypeMap = {
 export interface TagProps {
   /**
    * prop, used for setting type of the component from
-   * TagTypes - primary, primary-filled, positive, negative, warning, secondary, muted */
+   * TagTypes - primary, primary-filled, positive, negative, warning, secondary, muted, featured */
```

Comment:
Maybe we can just remove this list in the comment, it should show the options in your editor anyway and we don't really do this anywhere else.